src/sgame/sg_bot_skilltree.cpp

```diff
@@ -50,38 +53,58 @@ static std::set<std::string> G_ParseSkillsetList( Str::StringRef skillsCsv )
 	return skills;
 }
 
-static std::set<std::string>& G_GetDisabledSkillset()
+static std::set<std::pair<int, std::string>> G_ParseSkillsetListWithLevels( Str::StringRef skillsCsv )
 {
-	static std::set<std::string> disabledSkillset;
-	return disabledSkillset;
+	std::set<std::pair<int, std::string>> skills;
+
+	for (Parse_WordListSplitter i(skillsCsv); *i; ++i)
+	{
+		if ( strnlen(*i, 3) < 3 || !isdigit( (*i)[0] ) || (*i)[1] != ':' )
+		{
+			Log::Warn("Invalid \"int:skill\" pair: " QQ("%s"), *i);
+			continue;
+		}
+		int number = (*i)[0] - '0';
+		std::string skill = *i + 2;
+		skills.insert( { number, std::move(skill) } );
```

Comment:
It would change the semantics because currently you can have different skill names that unlock the same skill enum value (although it's not used in the code currently). I think it's good like this

Reply:
Parsing to a skillset is exactly what you would need for aliases to work *correctly*. G_SkillDisabled does a string comparison, so if you put one alias of a skill as disabled then it would not actually disable the skill.

Reply:
I mean I _wouldn't_ want different names to be fused to one value. It's not something that we rely on though, so I think it doesn't matter much for now.